### PR TITLE
Update CCApplication-win32.cpp

### PR DESCRIPTION
**Pull request by chizhanov**
Optimization

### CODE REVIEW

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
3>..\platform\win32\CCApplication-win32.cpp(97): error C2065: 'nFreq' : undeclared identifier
3>..\platform\win32\CCApplication-win32.cpp(97): error C2228: left of '.QuadPart' must have class/struct/union

File: cocos/platform/win32/CCApplication-win32.cpp [hunk 1]
```diff
@@ -94,7 +96,7 @@ int Application::run()
         }
         else
         {
-            Sleep(1);
+            Sleep((int)(((double)_animationInterval.QuadPart/(double)nFreq.QuadPart)*1000));
```
Comment (WenhaiLin) [inline cocos/platform/win32/CCApplication-win32.cpp:99]: This is not the correct logic.

**Comment by WenhaiLin** [comment]
@minggo This is not  the right solution.Please close this pr.

**Comment by WenhaiLin** [comment]
@andyque Please close this pr.

**Pull request closed without merging**